web/docs/operators/save.md

```diff
@@ -23,6 +23,11 @@ The `save` operator operates on raw bytes.
 Notably, it cannot be used after an operator that emits events, but rather only
 with operators that emit bytes, e.g., [`write`](write.md) or [`load`](load.md).
 
+### `--progress`
+
+Emit events describing the progress of the save operation. Note that this turns
+the `save` operator into a transformation rather than a sink.
+
```

Comment:
As suggested during the demo: I would recommend to remove this from the docs and from diagnostics.